docs/test_plan.rst

```diff
@@ -0,0 +1,155 @@
+LORE Test Plan
+==============
+
+LORE is known to render differently in Firefox and Chrome, with Chrome
+producing less satisfactory results. So testing on Chrome is
+recommended.
+
+#. .. rubric:: As Administrator...
+      :name: as-administrator...
+
+   1. .. rubric:: Repositories
+         :name: repositories
+
++-----------------------+----------+-------------------------------------------------------------------------+
+| Action                | Result   | Notes                                                                   |
++-----------------------+----------+-------------------------------------------------------------------------+
+| Create a repository   |          | You may only create a repository if you are a member of the '' group.   |
++-----------------------+----------+-------------------------------------------------------------------------+
++-----------------------+----------+-------------------------------------------------------------------------+
+
+Courses
+-------
+
+Expect a large course import to be slow, anywhere from several minutes
+to several hours.
```

Comment:
Might be useful to have an evidence-based estimate here for a specific course or courses.